new/PRS13727Salomon.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13727Salomon" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Salomon</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2022-02-17">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ሰሎሞን፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Salomon</persName>
+                    <floruit notBefore="1750" notAfter="1799"/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13727Salomon" passive="BLorient669"/>
+                    <relation name="lawd:hasAttestation" active="PRS13727Salomon" passive="EMML3621" cert="low">
+                        <desc>A Walda Gabrǝʾel and his son Salomon are attested in <ref type="mss" corresp="EMML3621"/>, dated
+                            to the eighteenth century as <ref type="mss" corresp="BLorient669"/>. Whether these persons are identical with those mentioned in <ref type="mss" corresp="BLorient669"/>
+                        can only be speculated.</desc>
```

Comment:
I would say, "is not clear yet". Theoretically, one would have some ways to check if they are the same persons, proviede both manuscripts are available.